makefiles/docker.inc.mk

```diff
@@ -70,6 +70,8 @@ export DOCKER_ENV_VARS += \
   TOOLCHAIN \
   UNDEF \
   WERROR \
+  WIFI_PASS \
+  WIFI_SSID \
```

Comment:
Are those defined and documented anywhere in a generic way? grepping through the RIOT repo only shows them being mentioned in esp-common and atwinc15x0 driver documentation, where they are expected to be passed within CFLAGS anyway.

So instead of adding them here, I would use the same approach as for the other Makefile variables just for `examples/gnrc_border_router`.

Reply:
They are actually not used by the example, but used by the only two WIFI drivers (ESP* and ATWINC15x0) we have.

They are documented separately in both drivers.

Reply:
I'm not sure I follow. The example defines the environment variables in https://github.com/RIOT-OS/RIOT/blob/master/examples/gnrc_border_router/Makefile#L19-L20 and uses them in https://github.com/RIOT-OS/RIOT/blob/master/examples/gnrc_border_router/Makefile.wifi.conf to add them to `CFLAGS`.

The documentation of both ESP* and the ATWINC15x0 explicitly adds them to `CFLAGS` as well (e.g., https://github.com/RIOT-OS/RIOT/blob/master/cpu/esp32/doc.txt#L1639). The only other example I could find that uses these environment variables is `examples/paho-mqtt` which also explicitly adds them to `CFLAGS` later on: https://github.com/RIOT-OS/RIOT/blob/master/examples/paho-mqtt/Makefile#L62-L63.

So my point was that the environment variables themselves are not generically defined, but only custom to specific examples, and therefore I think those examples should add the respective `DOCKER_ENV_VARS` themselves.

Reply:
@maribu ping

Might be nice to have this in before the hard feature freeze in two weeks from now.